### DISCUSSION TITLE
Old AE remains after changing main AE title

### DISCUSSION THREAD

**Discussion by marcheschi** [dcm4che/dcm4chee-arc-light#3276] [Q&A] [answered]
Hi, 
I'm sorry if this is an old question, but i cannot find in previous answers.
I'm on version: 5.23.3,54d7426,2021-05-17
I installed the dcm4chee-arc-light without security, and I sent some images to it .
Next I followed:
[https://github.com/dcm4che/dcm4chee-arc-light/wiki/AE-Title(s)-of-the-Archive](https://github.com/dcm4che/dcm4chee-arc-light/wiki/AE-Title(s)-of-the-Archive
)
to change the AE title of the archive to DCM4IEHR
now I have still the  DCM4CHEE  web app service in the main Studies page, when I select it and I submit I have  a
 Error 404
No such Application Entity: DCM4CHEE
Old images are still visible with "AS_RECEIVED"

In my opinion this DCM4CHEE must be substituted by the new one.

Thank you
Paolo

**Comment by vrindanayak** [comment, marked as answer]
If an application entity title is changed and this application entity is also used in RESTful services - which can be accessed using configured web applications, the changed AET value shall be updated also in `Web Service Path` and `AE Title` fields of the corresponding web applications. See [AE for Web Services](https://github.com/dcm4che/dcm4chee-arc-light/wiki/AE-Title(s)-of-the-Archive#ae-for-web-services) in [AE Titles of Archive](https://github.com/dcm4che/dcm4chee-arc-light/wiki/AE-Title(s)-of-the-Archive)

**Reply by marcheschi** [reply]
Hi Vrinda 
Thank you for your fast reply.
So I changed also from DCM4CHEE to DCM4IEHR in the webservices,
Now when I submit the DCM4IEHR in the main studies page I have still the same error:
Error 404
No such Application Entity: DCM4CHEE

DCM4CHEE is still somewhere.
Paolo

**Reply by vrindanayak** [reply]
Perhaps the old web application still seems to be cached in the browser or the configuration did not reload properly on saving the configuration. Could you please reload the configuration and the browser and retry once?
Also, the Web Service Path is changed to `/dcm4chee-arc/aets/DCM4IEHR/rs` right?

**Reply by marcheschi** [reply]
You where right, I forgot to change the Web service path.
Thank you very much.
Paolo
